src/JuMP.jl

```diff
@@ -88,6 +88,11 @@ type Model <: AbstractModel
     linconstrDuals::Vector{Float64}
     conicconstrDuals::Vector{Float64}
     constrDualMap::Vector{Vector{Int}}
+    # Vector of the same length as sdpconstr
+    # sdpconstrSym[i] is the list of pairs (i,j), i > j
```

Comment:
Yes indeed, the index should the constraint index, I will change that tomorrow :)